src/main/js/main_html5.js

```diff
@@ -873,11 +875,37 @@ require("../../../html5-common/js/utils/environment.js");
             lang: audioTracks[index].language,
             enabled: audioTracks[index].enabled
           };
+          if (audioTracks[index].enabled) {
```

Comment:
Do we need a null check for `audioTracks[index]`?

Reply:
No, it is loop. Index is index of a track, the track can be null.

Reply:
I know it's unlikely, but it's possible to have an array like:
```
[{}, null, {}, {}]
```
The check above would throw an error in this case. Even though the tracks are not supposed to contain nulls, I assume they are coming from external sources (i.e. Bitmovin, the video element, etc.) so we shouldn't trust that they will always have the expected format.